supertokens_python/recipe/session/cookie_and_header.py

```diff
@@ -62,23 +65,23 @@ def set_front_token_in_headers(
 
 
 def get_cors_allowed_headers():
-    return [ANTI_CSRF_HEADER_KEY, RID_HEADER_KEY]
+    return [
+        ANTI_CSRF_HEADER_KEY,
+        RID_HEADER_KEY,
+        AUTHORIZATION_HEADER_KEY,
+        AUTH_MODE_HEADER_KEY,
+    ]
 
 
 def set_header(response: BaseResponse, key: str, value: str, allow_duplicate: bool):
-    try:
-        if allow_duplicate:
-            old_value = response.get_header(key)
-            if old_value is None:
-                response.set_header(key, value)
-            else:
-                response.set_header(key, old_value + "," + value)
-        else:
+    if allow_duplicate:
+        old_value = response.get_header(key)
+        if old_value is None:
             response.set_header(key, value)
-    except Exception:
-        raise_general_exception(
-            "Error while setting header with key: " + key + " and value: " + value
-        )
+        else:
+            response.set_header(key, old_value + "," + value)
+    else:
+        response.set_header(key, value)
```

Comment:
cross check once if the frameworks themselves will append duplicate keys or not.

Reply:
All the response wrappers use `self.response.headers[key] = value` (so no scope of duplicates) and this function logic is using `get_header` as well. So there cannot be any issue.